### PR TITLE
Initial Android support

### PR DESCRIPTION
**Pull request by andr2000**

### CODE REVIEW

File: recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SUMMARY = "Android"
+DESCRIPTION = "Recipe for building Android image"
+
+LICENSE = "MIT"
```
Comment (lorc) [inline recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb]: Android is licensed under Apache license.

**Review by lorc — CHANGES_REQUESTED** [review]

File: recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb [hunk 2]
```diff
@@ -19,6 +19,9 @@ DEPENDS += "${JDK_VER} lz4-native"
 ANDROID_CCACHE = "prebuilts/misc/linux-x86/ccache/ccache"
 ANDROID_CCACHE_SIZE = "50G"
 
+ANDROID_JACK_ADMIN = "prebuilts/sdk/tools/jack-admin"
+ANDROID_JACK_MEM_SIZE = "4g"
```
Comment (lorc) [inline recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb]: Why `50G` but `4g` ? And why `4G` at all? Can you calculate it based on host RAM size?

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymy_babchuk@epam.com>`

**Review by arminn — APPROVED** [review]

File: recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+require inc/xt_shared_env.inc
+
+inherit java
+
+JDK_VER = "openjdk-8-native"
+
+DEPENDS += "${JDK_VER} lz4-native"
+
+ANDROID_PRODUCT ?= "aosp_arm64"
+ANDROID_VARIANT ?= "eng"
```
Comment (aanisov) [inline recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb]: Why eng build, not a userdebug f.e.?

**Review by aanisov — APPROVED** [review]

**Pull request merged**